src/Api/BarcodeApi.cs

```diff
@@ -87,12 +88,14 @@ public BarcodeApi(string clientSecret, string clientId)
         }
 
         /// <summary>
+        /// This method is obsolete and will be removed in next releases. Use new async method <see cref="GetBarcodeGenerateAsync" /> instead.
         ///     Generate barcode.
         /// </summary>
         /// <param name="request">Request. <see cref="GetBarcodeGenerateRequest" /></param>
         /// <returns>
         ///     <see cref="System.IO.Stream" />
         /// </returns>
+        [System.Obsolete]
```

Comment:
Add a message describing the replacement method
```suggestion
        [System.Obsolete("Message")]
```